bitnami/dataplatform-bp2/values.yaml

```diff
@@ -1110,15 +1231,13 @@ logstash:
       requests:
         cpu: 100m
         memory: 128Mi
+    ## @param logstash.metrics.service.clusterIP Static clusterIP or None for headless services
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#choosing-your-own-ip-address
     ## @param logstash.metrics.service.port Logstash Prometheus port
-    ## @param logstash.metrics.service.annotations [object] Annotations for the Prometheus metrics service
     ##
     service:
+      clusterIP: ""
       port: 9198
-      annotations:
```

Comment:
are the annotations present by default?

Reply:
yes Geeta, there are added in the Bitnami/logstash chart.